gradlew

```diff
@@ -105,8 +105,10 @@ if [ "$cygwin" = "false" -a "$darwin" = "false" -a "$nonstop" = "false" ] ; then
 fi
 
 # For Darwin, add options to specify how the application appears in the dock
+# In addition, we want to disable the MaxFDLimit for MacOS, which is set to a lower limit than the actual system maximum.
```

Comment:
```suggestion
# In addition, we want to increase the file descriptor limit to the MaxFDLimit in MacOS which, by default, is set to a lower limit than the actual system's maximum.
```

Reply:
To clarify, the JVM option `-XX:-MaxFDLimit` is to _disable_ the low `MaxFDLimit` set in JDK. By disabling, it can pick up the system FD limit, which is larger and proper. 

The suggested sentence looks incorrect.

Reply:
I noticed there are places where we enable the flag, and other places where we disable the flag. Is that intentional?

Reading this documentation: https://docs.oracle.com/cd/E19683-01/806-7930/filestructure-11/index.html it looks like setting the flag i.e. doing `-XX:+MaxFDLimit` should `Bump the number of file descriptors to max`. Isn't this what we are trying to accomplish here?

Reply:
No. The max FD constant for macOS is low. It is `OPEN_MAX == 10240` https://github.com/openjdk/jdk/blob/dfd3da3f52480f68f653beb1e720691f8232ace7/src/hotspot/os/bsd/os_bsd.cpp#L1997-L2002

You want to _disable_ the max FD, so that it can pick up the system's limit.